cmake/WindowsDllCopy.cmake

```diff
@@ -23,8 +30,9 @@ foreach(DLL_NAME ${DLLS})
     message(FATAL_ERROR "Unable to find dependency ${DLL_NAME}")
   endif()
 
-  message("Copying ${DLL_NAME} to ${DST}")
+  if($ENV{CI})
```

Comment:
This will always evaluate to false regardless of its value. Cmake ignores environment variables. You need to compare its value like `if($ENV{CI} MATCHES "true")`.

Reply:
> This will always evaluate to false regardless of its value.

I don't think that's correct.  [It worked in the CI][1].  But your way is better, as it doesn't need you to set CMP0012 policy.  Also it's better programmatically.  Updated.

[1]: https://github.com/neovim/neovim/actions/runs/4176134069/jobs/7232022075#step:6:901

Reply:
What the? You're right. Weird, I thought I tried this. My bad.